pkgs/by-name/qu/quick-sasl/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  pkg-config,
+  cmake,
+  arpa2cm,
+  arpa2common,
+  quickmem,
+  cyrus_sasl,
+  quickder,
+  libkrb5,
+  libev,
+  e2fsprogs,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quick-sasl";
+  version = "0.13.2";
+
+  src = fetchFromGitLab {
+    owner = "arpa2";
+    repo = "quick-sasl";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-kMKZRromm/hb9PZwvWAzmJorSqTB8xMIbWASfSjajiQ=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    arpa2cm
+    arpa2common
+    libkrb5
+    quickmem
+    cyrus_sasl
+    quickder
+    libev
+    e2fsprogs
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
```

Comment:
Maybe add a `mainProgram`? For me, it makes sense for it to be the server executable.